### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kalmanh**
1. Mention force_ssl -  a new option
2. Mention that it defaults to true
3. Show how to turn it off in development env...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +160,8 @@ RailsApiAuth.tap do |raa|
   raa.google_client_secret = '<your Google client secret>'
   raa.google_redirect_uri = '<your app redirect uri>'
 
+  # Force SSL for Oauth2Controller; defaults to `true`
+  raa.force_ssl = false if Rails.env.development?
```
Comment (marcoow) [inline README.md]: `raa.force_ssl = !Rails.env.development?`

Comment (kalmanh) [inline README.md]: My way is more expressive and easier on the brain :) ; your way is shorter. I am OK with either way - you tell me which one you want me to do.

Comment (marcoow) [inline README.md]: I'd prefer `raa.force_ssl = !Rails.env.development?` actually as `raa.force_ssl = false if Rails.env.development?` doesn't really feel idiomatic ;)

**Comment by marcoow** [comment]
replaced by #46

**Pull request closed without merging**